tests/phpunit/tests/kses.php

```diff
@@ -1496,4 +1496,255 @@ function test_wp_kses_main_tag_standard_attributes() {
 
 		$this->assertSame( $html, wp_kses_post( $html ) );
 	}
+
+	/**
+	 * Test that object tags are allowed under limited circumstances.
+	 *
+	 * @ticket 54261
+	 *
+	 * @dataProvider data_wp_kses_object_tag_allowed
+	 *
+	 * @param string $html     A string of HTML to test.
+	 * @param string $expected The expected result from KSES.
+	 */
+	function test_wp_kses_object_tag_allowed( $html, $expected ) {
+		$this->assertSame( $expected, wp_kses_post( $html ) );
+	}
+
+	/**
+	 * Data provider for test_wp_kses_object_tag_allowed().
+	 */
+	function data_wp_kses_object_tag_allowed() {
+		return array(
```

Comment:
What would you think of adding a test for `type="application/pdfa"` to ensure that starting with `application/pdf` isn't enough?

Reply:
👍🏻 6e2009c43f